scp/lib/timetable/pracSection.dart

```diff
@@ -193,8 +193,8 @@ storeSectionData(BuildContext context) async {
       MaterialPageRoute(builder: (context) => TimeTable()),
     );
   } else if (cardNumber == 1) {
-    Navigator.push(
-        context, MaterialPageRoute(builder: (context) => AttendanceTracker()));

```

Comment:
Will it never go in this block? If yes it will look like the app went unresponsive. If No, comment / remove that else if

Reply:
We removed the attendance tracer feature, and we aren't changing the cardNumber value on tap so the else if block is never executed I should have removed the bock.